docs/reference/search/profile.asciidoc

```diff
@@ -1051,7 +1051,7 @@ And here is the fetch profile:
             "load_source_count": 5
           },
           "debug": {
-            "stored_fields": ["_id", "_routing", "_source"]
+            "stored_fields": ["_id", "_ignored", "_routing", "_source"]
```

Comment:
This is a consequence of exposing the correct stored fields spec in FetchFieldsPhase, that take stored metadata fields into account. _ignored will be removed again once it's no longer stored. The problem is that the field should have been there since its introduction, but it was never added to `StoredFieldLoader#fieldsToLoad` which is where the other three fields come from. Note that `StoredFieldsPhase` did not include in its stored fields spec the default metadata fields that it always requested.

For posterity, why is `_type` not there if fetch fields phase requests it by default? Because it is not mapped in recent clusters, and it is only part of the stored fields spec, hence loaded, when it is mapped, which is the case only in very old archive indices.

Reply:
:+1: